### PR TITLE
Add version as tag to internal metrics.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: test/test_client.py [hunk 1]
```diff
@@ -29,6 +30,20 @@ def setUp(self):
...
+        with patch.object(
+                registry,
+                'WavefrontSdkMetricsRegistry',
+                return_value=no_registry) as mock_registry:
+            WavefrontClient(
+                'no_server',
+                'no_token',
+                flush_interval_seconds=86400,
+                enable_internal_metrics=True)
+        self.assertTrue('version' in mock_registry.call_args[1]['tags'])
```
Comment (oppegard) [inline test/test_client.py:45]: I assume we have an actual value for the `version` tag? If so, can we assert that it matches a semver regex? For example: `^v\d+\.\d+\.\d+$`

**Review by oppegard — CHANGES_REQUESTED** [review]
Requested a small test improvement.

File: test/test_client.py (hunk 1 above)
Comment (keep94) [reply test/test_client.py]: While the semver can be fetched when I run the unit tests locally, fetching the semver on github fails so on github version resolves to **unknown**.  Therefore I tweaked the regex you provided to allow for either **unknown** or a semver.

**Review by oppegard — APPROVED** [review]
lgtm

**Pull request merged**